sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/FieldValueTypeInformation.java

```diff
@@ -125,18 +126,20 @@ public static FieldValueTypeInformation forOneOf(
         .build();
   }
 
-  public static FieldValueTypeInformation forField(Field field, int index) {
-    TypeDescriptor<?> type = TypeDescriptor.of(field.getGenericType());
+  public static FieldValueTypeInformation forField(
+      Field field, int index, Map<Type, Type> boundTypes) {
```

Comment:
In this case, I think so. This is a logically an internal class, which was only marked public because we needed to access it from other packages (e.g. the protobuf package). The fact that it was not marked @Internal was likely a mistake, and I think it's fine to change it in this case.